src/constants.js

```diff
@@ -0,0 +1,12 @@
+// @flow
+
+declare var __DEV__: ?string
+
+export const SC_ATTR = 'data-styled-components'
+export const CONTEXT_KEY = '__styled-components-stylesheet__'
+
+export const IS_BROWSER = typeof window !== 'undefined'
+
+export const DISABLE_SPEEDY =
```

Comment:
What if we made this an instance variable of the StyleSheet such that it could be overridden if desired by an external composer?

Reply:
Let’s discuss this separately as well. We have another issue where we were discussing to add it to the StyleSheetManager. We previously have been against exposing any kind of global function/flags to change settings like these though